certificate-service/domain/src/test/java/se/inera/intyg/certificateservice/domain/validation/model/ElementValidationDateRangeListTest.java

```diff
@@ -930,6 +1024,99 @@ void shouldReturnNoValidationErrorsIfCorrectPeriodsWithNoOverlap() {
 
       assertEquals(Collections.emptyList(), actualResult);
     }
+
+    @Nested
+    class BeforeMinDate {
```

Comment:
Please also add some tests to verify behaviour when min is not set.

For example, take a look at ```ElementValidationTextTest``` that handles both mandatory and limit tests.